eng/push-tarball.sh

```diff
@@ -129,6 +129,16 @@ pushd "$vmr_path"
     git push -u upstream "$new_branch_name"
 
     echo "Creating PR from $new_branch_name to $target_branch"
-    curl -H 'Content-Type: application/json' -d "$data" "$pr_url"
+    curlResult=$(curl -H 'Content-Type: application/json' -d "$data" "$pr_url" 2>/dev/null)
+    webUrl=$(jq -r '.repository.webUrl | values' <<< "$curlResult")
+    pullRequestId=$(jq -r '.pullRequestId | values' <<< "$curlResult")
+    if [ -n "$webUrl" ] && [ -n "$pullRequestId" ]; then
+      echo "$webUrl"/pullrequest/"$pullRequestId"
+    elif [[ "$curlResult" == *"TF401179"* ]]; then
+      "$(jq -r '.message' <<< "$curlResult")"
+    else
+      echo "##vso[task.logissue type=error]An unexpected error has occurred during the pr request!"
```

Comment:
```suggestion
      echo "##vso[task.logissue type=error]An unexpected error has occurred while creating the security partners PR!"
```

Reply:
Can you think of including some useful information or details in the error (e.g., the url or the PR id?)

Reply:
I think the PR id and url are available after the PR is created successfully. Additionally, the data that was used to create the PR was previously printed. I am attempting to print the request result, which I presume has all the information, in the following line: But I am really open to suggestions.

Reply:
The whole response is printed below so I'd just leave it at that.